src/contracts/interfaces/IPool.sol

```diff
@@ -185,13 +185,29 @@ interface IPool {
     uint256 variableBorrowIndex
   );
 
+  /**
+   * @dev Emitted when the deficit of a reserve is covered.
+   * @param reserve The address of the underlying asset of the reserve
+   * @param caller The caller that triggered the DeficitCovered event
+   * @param amountCovered The amount of deficit covered
+   */
+  event DeficitCovered(address indexed reserve, address caller, uint256 amountCovered);
```

Comment:
does it make sense to make `caller` indexed?

Reply:
It's the address calling the function and in effect burning it's aTokens / GHO - it's a permissioned entity registered as UMBRELLA on the addresses provider.

I don't see the point in making it indexed though.

Reply:
In case there are multiple Umbrella instances could be useful. If gas costs are not relevant I may be a good idea to add it.